Source/WebCore/page/Page.cpp

```diff
@@ -1772,7 +1777,8 @@ void Page::updateRendering()
     // This function is not reentrant, e.g. a rAF callback may trigger a forces repaint in testing.
     // This is why we track m_renderingUpdateRemainingSteps as a stack.
     if (m_renderingUpdateRemainingSteps.size() > 1) {
-        layoutIfNeeded();
+        WTFLogAlways("Recursed into updateRendering, flushing and returning");
```

Comment:
Do we want this? Does this fit in with our other uses of `WTFLogAlways`?

I don’t think we’d want this log to be landed unconditionally if it’s going to happen during normal operation.

Reply:
We definitely do not, just a remnant of my debugging process :)